src/Appwrite/Utopia/Response/Model/Membership.php

```diff
@@ -22,24 +22,30 @@ public function __construct()
                 'default' => '',
                 'example' => '5e5ea5c16897e',
             ])
+            ->addRule('userName', [
+                'type' => self::TYPE_STRING,
+                'description' => 'User name.',
+                'default' => '',
+                'example' => 'John Doe',
+            ])
+            ->addRule('userEmail', [
+                'type' => self::TYPE_STRING,
+                'description' => 'User email address.',
+                'default' => '',
+                'example' => 'john@appwrite.io',
+            ])
             ->addRule('teamId', [
                 'type' => self::TYPE_STRING,
                 'description' => 'Team ID.',
                 'default' => '',
                 'example' => '5e5ea5c16897e',
             ])
-            ->addRule('name', [
+            ->addRule('teamName', [
```

Comment:
Any reason why not to add teamID too? Could not be useful for us in UI, but might be really useful in server-SDK. Overall, if we mention some resources (like a team) and provide some specific data (name), we should also provide ID so they can get all other data of the team if they need to.